hack/scripts/provision.sh

```diff
@@ -341,6 +342,25 @@ write_flintlockd_config() {
 	local parent_iface="$2"
 	local bridge_name="$3"
 	local insecure="$4"
+	local auth_file="$5"
+
+	if [[ -n "$auth_file" ]]; then
+	  say "using auth file: $auth_file"
+    # shellcheck source=user-defined
+    . "${auth_file}"
+    if [[ -n "$basic_auth_token" ]]; then
+      auth_information="basic-auth-token=$basic_auth_token"
+    fi
+    if [[ -n "$tls_cert" ]]; then
+      # shellcheck disable=SC2154
+      auth_information=<< TLS
+tls-cert="$tls_cert"
+tls-key="$tls_key"
+tls-client-validate="$tls_client_validate"
+tls-client-ca="$tls_client_ca"
+TLS
+    fi
+	fi
```

Comment:
can you run a formatter over this? the spacing is very weird

Reply:
:nod: will do.